### ISSUE TITLE
Error attempting to install package in spacemacs

### ISSUE THREAD

**Issue by Zonr0** [masayuko/ox-rst#46]
I'm a bit of a newcomer to the emacs ecosystem, so please forgive me if this is just user error.

Attempting to install the package in spacemacs results in the following error:
``An error occurred while installing ox-rst (error: (error Package ‘emacs-26.1’ is unavailable))``

I have tried in the user configuration section of .spacemacs
1. (use-package ox-rst :ensure t)
2. (use-package ox-rst :ensure nil)
3. (require 'ox-rst)

I have also tried (what feels like the more idiomatically correct spacemacs method) of adding ox-rst to the `dotspacemacs-additional-packages` function in .spacemacs.

**Comment by msnoigrs** [comment]
It seems that Spacemacs is equivalent to emacs 24.4 or 25.2. (I'm not a spacemacs user)
Ox-rst requires emacs 26.1 or above.

Please see also this issue https://github.com/syl20bnr/spacemacs/issues/10796

**Comment by Zonr0** [comment]
To follow up on this for anyone curious:

Turns out the windows binaries linked on the spacemacs repo are pretty out of date (2-3 years on the latest snapshots) and seem to come from a dead project. I grabbed the 26.3 official binaries linked from the [official emacs page](https://www.gnu.org/software/emacs/download.html) and that resolved my issue. I haven't noticed any breaking changes with spacemacs in doing so.

Upgrading is straightforward and works pretty much exactly as you expect. I basically just backed up my old emacs folder, extracted the emacs-26.3-x86_64.zip archive and renamed it to emacs.
